third_party/color_emoji/emoji_builder.py

```diff
@@ -179,12 +179,15 @@ def write_format1 (self, png):
 				self.write (pixel)
 			offset += stride
 
-	png_allowed_chunks =  ["IHDR", "PLTE", "tRNS", "sRGB", "IDAT", "IEND"]
+	png_allowed_chunks =  [
+		"IHDR", "PLTE", "tRNS", "sRGB", "IDAT", "IEND", # Python2
+		b"IHDR", b"PLTE", b"tRNS", b"sRGB", b"IDAT", b"IEND", # Python3
```

Comment:
Just use bytes for both python2 and 3 in this case (also see me comment below about using BytesIO instead of StringIO).